package.json

```diff
@@ -13,6 +13,7 @@
     "typecheck": "graphql-codegen && tsc --noEmit --pretty",
     "lint:check": "eslint . --max-warnings=0",
     "lint:fix": "eslint . --fix",
+    "lint:tsdoc": "eslint -f unix \"src/**/*.{ts,tsx}\"",
```

Comment:
Why does this need a seperate script? The `lint:fix` and `lint:check` do the same thing.